platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -451,9 +451,16 @@ private extension WysiwygComposerViewModel {
             let attributed = try HTMLParser.parse(html: html,
                                                   style: parserStyle,
                                                   mentionReplacer: mentionReplacer)
-            // FIXME: handle error for out of bounds index
             let htmlSelection = NSRange(location: Int(start), length: Int(end - start))
-            let textSelection = try attributed.attributedRange(from: htmlSelection)
+            let textSelection: NSRange
+
+            // Workaround for the "out of bounds index" error
```

Comment:
Given we think this workaround is needed because the rust model is outputting invalid selection indices, perhaps we can create an issue for that and link it here?

Reply:
Here -> https://github.com/matrix-org/matrix-rich-text-editor/issues/818

Reply:
Sorry - I meant to add a link to the issue in the code comment itself - as it might help whoever runs into this workaround later. But it doesn't matter too much, given they can probably find this PR.